src/main/webapp/WEB-INF/view/photos.jsp

```diff
@@ -38,13 +40,13 @@ Conversation conversation = (Conversation) request.getAttribute("conversation");
   <nav>
     <a id="navTitle" href="/">IMhere!</a>
     <a href="/conversations">Conversations</a>
+    <a href="/about.jsp">About</a>
+    <a href="/activity-feed">Activity Feed</a>
     <% if(request.getSession().getAttribute("user") != null){ %>
-      <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+      <a href="/notifications">Notifications</a>
```

Comment:
I think there's a conflict here, so when you resolve make sure to keep the version of the nav bar that's on master

Reply:
sounds good!